Resources/public/mapbender.element.digitizer.js

```diff
@@ -1285,42 +1266,68 @@
         /**
          * Copy feature
          *
-         * @param feature
+         * @param {OpenLayers.Feature} feature
          */
-        copyFeature: function(feature) {
+        copyFeature: function(feature ) {
             var widget = this;
             var schema = feature.schema;
+            var layer = schema.layer;
             var newFeature = feature.clone();
             var config = schema.copy;
             var defaultAttributes = getValueOrDefault(config, "data",{});
+            var allowCopy = true;
+
+            _.each(schema.copy.rules, function(ruleCode) {
+                var f = feature;
+                eval('allowCopy = ' + ruleCode + ';');
```

Comment:
Should be ```allowCopy = allowCopy && <new expression>;``` I think. That way, if you have multiple rules, copying is only allowed if all rules pass.

edit: nvm, you use the early exit from _.each by returning false

Reply:
> copying is only allowed if all rules pass.

Exactly! The evaluated code should return a value to be assigned by `allowCopy` to check if it's false, then stop to check.